.github/workflows/build-and-test.yml

```diff
@@ -198,3 +198,49 @@ jobs:
           file: ./coverage.txt
           fail_ci_if_error: true
           verbose: true
+  cross-build-collector:
+    needs: [setup-environment]
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
```

Comment:
That makes sense. I have updated the matrix to match the opentelemetry-collector-releases matrix. There is one difference and that is the Darwin 386 build is no longer supported by Go. This is because it dropped support for 32-bit binaries on macOS in [Go 1.15](https://go.dev/doc/go1.15).

Reply:
Thanks for catching that, created open-telemetry/opentelemetry-collector-releases/issues/97 to track this